content/influxdb/v2.2/api-guide/tutorials/client-library-starter/python.md

```diff
@@ -0,0 +1,720 @@
+
+    # Activate the virtualenv (OS X & Linux)
+    source virtualenv/bin/activate
+    ```
+
+3. After activation completes, enter the following commands in your terminal to install Flask and Jinja with the `pip` package installer (included with Python):
+
+    ```bash
+    pip install Flask
+    pip install Jinja
+    ```
+
+4. In your project, create a `app.py` file that:
+
+   1. Imports the Flask package.
+   2. Instantiates a Flask application.
+   3. Provides a route to execute the application.
+
+   ```python
+   from flask import Flask
+   app = Flask(__name__)
+
+   @app.route("/")
+   def hello():
+     return "Hello World!"
+   ```
+
+   {{% caption %}}[@influxdata/iot-python-app app.py](https://github.com/influxdata/iot-api-python/blob/main/app.py){{% /caption %}}
+
+   To start your application with the debugger and hot-reloader enabled,
+   enter the following command in your terminal:
+
+   ```bash
+   export FLASK_ENV=development
+   flask run
+   ```
+
+   In your browser, visit <http://localhost:5000> to view the “Hello World!” response.
+
+## Install InfluxDB client library
+
+The InfluxDB client library provides the following InfluxDB API interactions:
+
+- Query data with the Flux language.
+- Write data to InfluxDB.
+- Batch data in the background.
+- Retry requests automatically on failure.
+
+Enter the following command into your terminal to install the client library:
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[Python](#python)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+
+```bash
+pip install influxdb-client
+```
+
+For more information about the client library, see the [influxdata/influxdb-client-python repo](https://github.com/influxdata/influxdb-client-python).
+
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+## Configure the client library
+
+InfluxDB client libraries require configuration properties from your InfluxDB environment.
+Typically, you'll provide the following properties as environment variables for your application:
+
+- `INFLUX_URL`
+- `INFLUX_TOKEN`
+- `INFLUX_ORG`
+- `INFLUX_BUCKET`
+- `INFLUX_BUCKET_AUTH`
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[Python](#python)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+
+To set up the client configuration, create a `config.ini` in your project's top
+level directory and paste the following to provide the necessary InfluxDB credentials:
+
+```ini
+[APP]
+INFLUX_URL = <INFLUX_URL>
+INFLUX_TOKEN = <INFLUX_TOKEN>
+INFLUX_ORG = <INFLUX_ORG_ID>
+INFLUX_BUCKET = iot_center
+INFLUX_BUCKET_AUTH = iot_center_devices
+```
+
+{{% caption %}}[/iot-api-python/config.ini](https://github.com/influxdata/iot-api-python/blob/main/config.ini){{% /caption %}}
+
+Replace the following:
+
+- **`<INFLUX_URL>`**: your InfluxDB instance URL.
+- **`<INFLUX_TOKEN>`**: your InfluxDB [API token](#authorization) with permission to query (_read_) buckets
+and create (_write_) authorizations for devices.
+- **`<INFLUX_ORG_ID>`**: your InfluxDB organization ID.
```

Comment:
Can these be read from the environment rather than hardcoding them in the `.ini`?